docs/how-to-guides/integrating-autoware/tuning-parameters-and-performance/important-temporal-performance-metrics.md

```diff
@@ -0,0 +1,121 @@
+| AWOV-004 | **Response time** from CPA #0 to CPA #6              | Duration to process pointcloud data in Sensing and Detection.                                                    | Perception            | N/A             | Tracking relies on detection to provide real-time and up-to-date sensed data for accurate tracking.             | The metric is used if delay compensation is enabled in Tracking.   |
+| AWOV-005 | **Message rate** from CPA #4 to CPA #5               | Update rate of Detection result received by Tracking.                                                            | Perception            | 10 Hz           | Tracking relies on detection to provide real-time and up-to-date sensed data for accurate tracking.             |                                                                    |
+| AWOV-006 | **Response time** from CPA #0 to CPA #14             | Response time from output of observed data from LiDARs to its consumption in EKF Localizer via NDT Scan Matcher. | Localization          | N/A             | EKF Localizer relies on fresh and up-to-date observed data from sensors for accurate estimation of self pose.   |                                                                    |
+| AWOV-007 | **Message rate** from CPA #11 to CPA #13             | Update rate of pose estimated by NDT Scan Matcher.                                                               | Localization          | 10 Hz           | EKF Localizer relies on fresh and up-to-date observed data from sensors for accurate estimation of self pose.   |                                                                    |
+| AWOV-008 | **Message rate** from CPA #15 to CPA #12             | Update rate of feed backed pose estimated by EKF Localizer.                                                      | Localization          | 50 Hz           | NDT Scan Matcher relies on receiving estimated pose from EKF Localizer smoothly for linear interpolation.       |                                                                    |
+| AWOV-009 | **Message rate** from CPA #17 to CPA #19             | Update rate of Localization result received by Planning.                                                         | Localization          | 50 Hz           | Planning relies on Localization to update the estimated pose frequently.                                        |                                                                    |
+| AWOV-010 | **Response time** from CPA #20 to CPA #23            | Processing time from beginning of Planning to consumption of Trajectory message in Control.                      | Planning              | N/A             | The vehicle relies on Planning to update trajectory within a short time frame to achieve safe driving behavior. |                                                                    |
```

Comment:
"The vehicle" sounds the very specific car like "Lexus". "Control" or "Vehicle"?